otava/analysis.py

```diff
@@ -259,13 +247,13 @@ def split(series: np.array, window_len: int = 30, max_pvalue: float = 0.001,
     assert "Window length must be at least 2", window_len >= 2
     start = 0
     step = int(window_len / 2)
-    indexes = []
+    change_points = []
```

Comment:
Perhaps update the above comment, and check other cmments too. (grep mongodb and signal)